web/src/components/Inputs.js

```diff
@@ -13,15 +13,19 @@ const makeInput = InputInner => {
     label,
     hideLabel,
     type
-  }) => (
-    <div className="mb2">
-      <label htmlFor={name} className={hideLabel ? 'sr-only' : ''}>
-        {label}
-      </label>
-      <InputInner id={name} type={type} {...rest} />
-      {touched && error && <span>{error}</span>}
-    </div>
-  );
+  }) => {
+    const hasError = touched && error;
```

Comment:
So, ignoring validation errors until the field has been focused and then blurred?  Makes sense, just making sure I understand.  😛

Reply:
👍